### PR TITLE
Update snips.markdown

### PR DESCRIPTION
**Pull request by todschmidt**
**Description:**
 - Added probaility threshold and feedback sounds configuration
 - Added new services

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#13746

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/snips.markdown [hunk 1]
```diff
@@ -105,6 +105,12 @@ By default, Snips runs its own MQTT broker. But we can also tell Snips to use an
 
 ## {% linkable_title Home Assistant configuration %}
 
+Configuration variables:
```
Comment (frenck) [inline source/_components/snips.markdown:108]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/snips.markdown [hunk 2]
```diff
@@ -9,7 +9,7 @@ sharing: true
 footer: true
 logo: snips.png
 ha_category: Voice
-ha_release: 0.48
+ha_release: 0.66
```
Comment (frenck) [inline source/_components/snips.markdown]: This should not be updated. Snips support was added in `0.48`.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
@tschmidty69 I've reviewed this PR and left you some comments.
Could you please take a look? 👍

**Comment by balloob** [comment]
@tschmidty69 this PR did not make the 0.68 release but should have, as the support has gone out. Please fix it asap.

**Comment by todschmidt** [comment]
This is outdated

**Pull request closed without merging**